packages/cloud-cognitive/src/components/TagSet/TagSetModal.js

```diff
@@ -53,25 +58,31 @@ export const TagSetModal = ({
   };
 
   return (
-    <Modal
-      className={`${blockClass}__show-all-modal`}
+    <ComposedModal
+      className={`${blockClass}__show-all-tags-modal`}
       open={open}
       passiveModal
       size="sm"
-      modalHeading={heading}
-      onRequestClose={onClose}>
-      <Search
-        data-modal-primary-focus
-        className={`${blockClass}__show-all-tags-search`}
-        labelText={searchLabel}
-        placeholder={searchPlaceholder}
-        onChange={handleSearch}
-        size="lg"
-      />
-      <div className={`${blockClass}__show-all-tags-content`}>
-        {filteredModalTags}
-      </div>
-    </Modal>
+      onClose={onClose}>
```

Comment:
as a convention we should try to use spread for props with the same name. for example `...{ onClose, open }`

Reply:
Done